core/assets/environment_variables/production.json

```diff
@@ -1,4 +1,8 @@
 {
   "AVAILABLE_LOCALES": "en",
-  "DEFAULT_LOCALE": "en"
+  "DEFAULT_LOCALE": "en",
+  "FALLBACK_LOCALES": "en",
+  "MAILER_DEFAULT_HOST": "localhost",
+  "MAILER_DEFAULT_PORT": "80",
+  "APP_PORT": "3000"
```

Comment:
we can sort variables alphabetical, as we are usually doing for different parts of the code

Reply:
It was sorted in 57d5867 ✅ Btw, it will be updated in new version by later commit.